ccd-definition/AuthorisationCaseField/judiciary.json

```diff
@@ -159,5 +159,19 @@
     "CaseFieldID": "courtDirectionsCustom",
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "nextHearingDateList",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Assuming this label needs "CRU" as it's updated in the controller?

Reply:
Yeah so judge is going to create and update this label on the submission event. So it needs 'CRU'